__tests__/getGroupRDF.test.js

```diff
@@ -6,31 +6,32 @@ import * as getGroupRDF from '../src/getGroupRDF'
 import SinopiaServer from 'sinopia_server'
 import Honeybadger from 'honeybadger'
 
+const emptyHttpResult = { response: { body: { } } }
```

Comment:
Instead of returning `null`, return an empty http result, which is what happens with the non-mocked server. This lets our tests test actual behavior.